core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -516,6 +517,54 @@ public static String decodeToken(String token) throws S3Exception {
     }
   }
 
```

Comment:
why not get the all children needed from liststatuspartial call and then build the bucketresult?

Reply:
There is a filter when building ListBucketResult, It may affect the number of result sets, so it can not obtain all the results in one liststatuspartial.